plan-for-extension-exercise/src/planForExtensionExercise/Shape.java

```diff
@@ -0,0 +1,9 @@
+package planForExtensionExercise;
+
+public abstract class Shape {
```

Comment:
Because this abstract class only has methods, it could be an interface instead of an abstract class. Tracy, can you comment on why you made it an abstract class instead of an interface?